RevenueCatUI/Views/LoadingPaywallView.swift

```diff
@@ -71,6 +71,9 @@ private extension LoadingPaywallView {
         },
         restorePurchases: {
             fatalError("Should not be able to purchase")
+        },
+        trackEvent: { _ in
+            // Ignoring events from loading paywall view
```

Comment:
I love that the compiler made me implement this. And realized this was the best solution.